webapp/src/components/sidebar_buttons/sidebar_buttons.jsx

```diff
@@ -152,6 +154,21 @@ export default class SidebarButtons extends React.PureComponent {
                         {' ' + yourPrs.length}
                     </a>
                 </OverlayTrigger>
+
+                <OverlayTrigger
+                    key='githubMentionsLink'
+                    placement={placement}
+                    overlay={<Tooltip id='mentionTooltip'>{'Mentions on Pull Requests'}</Tooltip>}
+                >
+                    <a
+                        onClick={() => this.openRHS(RHSStates.MENTIONS)}
+                        style={button}
+                    >
+                        <i className='fa fa-comment-o'/>
+                        {' ' + mentions.length}
```

Comment:
I'm leaning towards not showing the number of mentions. 

There are users who are keen on keeping their inboxes empty. Given that they have no control over this number and can't reduce it, I wonder if we should omit the number. The button would still allow the user to check in on their conversation but would no longer nag them.

Reply:
That's a good point. I think it makes sense to remove the number. Would we then want to change the ordering of the mentions icon? I was thinking since it would be just an icon we can move it next to the refresh button

Reply:
It would look something like this:

![image](https://github.com/mattermost/mattermost-plugin-github/assets/32754336/9ad5feb5-362c-4b93-b581-f359f95e4b60)

Reply:
@hanzei Is it possible to "remove" mentions? Maybe unsubscribing to the issue/PR does this?

Reply:
@cyrusjc Do you know?

Reply:
> It would look something like this:
> 
> ![image](https://private-user-images.githubusercontent.com/32754336/328336360-9ad5feb5-362c-4b93-b581-f359f95e4b60.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTUxMTY5ODYsIm5iZiI6MTcxNTExNjY4NiwicGF0aCI6Ii8zMjc1NDMzNi8zMjgzMzYzNjAtOWFkNWZlYjUtMzYyYy00YjkzLWI1ODEtZjM1OWY5NWU0YjYwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MDclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTA3VDIxMTgwNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTVmMjg1Yzc0YTgzMzk1MWVhYzcwN2JhMWZlZDJmOWZlODRiMTc3ODZiZTlmM2M0NWQzZmFhYjFlZjk3Yzk2ZGQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.146mRu7kW1ed29FGwf4nZXvSijHq54sd5tcc-LZnt-k)

I like the suggestion :+1:

Reply:
I don't sorry.